source/wp-content/themes/wporg-showcase-2022/src/style/style.scss

```diff
@@ -145,3 +145,15 @@
 		}
 	}
 }
+
+.page-template-page-screenshot-auditor {
+	.wp-block-wporg-site-screenshot {
+		img {
+			border: 2px solid var(--wp--preset--color--blueberry-1);
```

Comment:
I chose the manual upload as the exception, but given how many sites have issues with mShots maybe we want to reverse this?